### PR TITLE
NEW add weighted amount on project/opportunity

### PR DESCRIPTION
**Pull request by lvessiller-opendsi**
NEW add weighted amount on project/opportunity

### CODE REVIEW

**Comment by lvessiller-opendsi** [comment]
Travis error doesn't concern this pull request.

File: htdocs/core/lib/project.lib.php [hunk 1]
```diff
@@ -1862,7 +1862,7 @@ function print_projecttasks_array($db, $form, $socid, $projectsListId, $mytasks
 		if (!empty($conf->global->PROJECT_USE_OPPORTUNITIES))
 		{
 			print_liste_field_titre("OpportunityAmount", "", "", "", "", 'align="right"', $sortfield, $sortorder);
-			print_liste_field_titre("OpportunityStatus", "", "", "", "", 'align="right"', $sortfield, $sortorder);
+			print_liste_field_titre('OpportunityWeightedAmount', '', '', '', '', 'align="right"', $sortfield, $sortorder);
```
Comment (eldy) [inline htdocs/core/lib/project.lib.php]: You remove the status of opportunity from columns ?

**Comment by lvessiller-opendsi** [comment]
You had an amount in this column instead of status
Opportunity_Index_Before

So I replace with weighted amount (and you have a little table and it will be hard to add another column).
Opportunity_Index_After

**Comment by philippe-opendsi** [comment]
@eldy the status is on the end of the line and in fact is useless because only open project are displayed

**Comment by eldy** [comment]
The status of project ("open" / "closed") is different than the status of opportunity that is "prospection", "qualification", "negociation", "win", "lost", ... It can be edited in dictionary.
My understanding is that you sacrifice the prospection status to replace it with weighted amount. Why not, but may be it is possible to keep the label of prospection status as a tooltip "?" after the amount ?

**Comment by philippe-opendsi** [comment]
@eldy ok didn't understand of wich status you were talking :-) 
Maybe we can replace in the table the status of the project wich is useless with the status of the opportunities. We try that...

**Comment by lvessiller-opendsi** [comment]
@eldy 
I replaced project status column by opportunity status column

**Comment by lvessiller-opendsi** [comment]
@eldy 
I think I put all wanted columns.
Could you check this PR ? (tell me if you have some remarks)

**Pull request merged**